cloudflare/resource_cloudflare_zone.go

```diff
@@ -47,6 +47,12 @@ func resourceCloudflareZone() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"plan": {
+				Type:         schema.TypeString,
+				Computed:     true,
```

Comment:
I don't think we want the computed option here. We have the ability to set this in the resource which negates the need for computing it later (such as a UUID or interpolated values)

```suggestion
```